model/build.gradle.kts

```diff
@@ -39,6 +39,19 @@ spine {
     enableJava()
 }
 
+configurations {
+    create("test")
+}
+
+tasks.register<Jar>("testArchive") {
+    archiveBaseName.set("module-test")
+    from(project.the<SourceSetContainer>()["test"].output)
+}
+
+artifacts {
+    add("test", tasks["testArchive"])
```

Comment:
Let's not repeat this string literal as-is. Can we somehow have a reference to previously registered task?